tests/test_foreign.py

```diff
@@ -85,6 +85,14 @@ def get_list_of_lists(result):
         result = list(prolog.query("get_list_of_lists(Result)"))
         self.assertTrue({'Result': [[1], [2]]} in result, 'Nested lists should be unified correctly as return value.')
 
+    def test_dictionary(self):
+        prolog = Prolog()
+        result = list(prolog.query("X = dict{value1:value1 , value2: value2}"))
+        dict = result[0]
+        print(dict)
```

Comment:
I think this line can be removed.

Reply:
Removed print statement, I don't know how I missed that